OracleWebCenterContent/README.md

```diff
@@ -1,14 +1,15 @@
-# Oracle WebCenter Content Container Image
+# Oracle WebCenter Content container images
 
-This project offers scripts to build an Oracle WebCenter Content container image based on 12.2.1.4.0 release. Use this documentation to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
+This project offers scripts to build an Oracle WebCenter Content container image based on 12.2.1.4.0 release. Use this documentation to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, please refer to the Oracle WebCenter Content 12.2.1.4.0 Online Documentation at [Oracle Fusion Middleware](https://docs.oracle.com/en/middleware/fusion-middleware/), specifically the section on
+- Oracle WebCenter Content
 
-## For Building & Running the WebCenter Content Container images and Using
+## Building, running and using the WebCenter Content container images
 
 **IMPORTANT**
 - Refer [Oracle WebCenter Content 12.2.1.4.0](dockerfiles/README.md) for detailed set up.
 
 ## License
-To download and run the WebCenter Content, regardless whether inside or outside a container, and regardless of the distribution, you must download the binaries from the Oracle content and accept the license indicated on that page.
+To download and run the WebCenter Content, regardless whether inside or outside a container, and regardless of the distribution, you must download the binaries from the Oracle website and accept the license indicated on that page.
```

Comment:
Please replace "Oracle website" with a link to the Oracle Software Delivery Cloud.